### DISCUSSION TITLE
Should we add YouTube channel or courses Links into our Hub?

### DISCUSSION THREAD

**Discussion by rupali-codes** [rupali-codes/LinksHub#126] [Ideas]
We are thinking to add a new `Category` for `YouTube` tutorials and `SubCategories` would be like `web developement` `frontend` `backend` `javascript` etc.

What are you takes on this? any suggestions?

**Comment by jayk-gupta** [comment]
Should we also add websites and text resources for learning CSS and JavaScript?

**Reply by rupali-codes** [reply]
yes we can surely

**Reply by jayk-gupta** [reply]
I would love to work on this and add resources for these.

**Reply by rupali-codes** [reply]
> I would love to work on this and add resources for these.

Thanks, you can go ahead

**Reply by jayk-gupta** [reply]
okay , I will work on it.
I am a beginner in opensource , so If any mistake I make kindly let me know.
Thanks!

**Reply by CBID2** [reply]
> okay , I will work on it.
> I am a beginner in opensource , so If any mistake I make kindly let me know.
> Thanks!

Here’s a [post](https://chrissycodes.hashnode.dev/how-to-make-a-pull-request-without-pulling-your-hair-out) I created for beginners in PRs/Open Source. Hope it helps @jayk-gupta! 😄

**Comment by CBID2** [comment]
> We are thinking to add a new `Category` for `YouTube` tutorials and `SubCategories` would be like `web developement` `frontend` `backend` `javascript` etc.
> 
> What are you takes on this? any suggestions?

That’s a good idea! It can make things more organized

**Comment by ujjawaltyagii** [comment]
@jayk-gupta  I have added the new category with name ``Languages`` in sidemenu, now you can add resources there in different sub-categories !
And feel free to ask anything 😊

**Comment by jayk-gupta** [comment]
Yeah, I added two javascript resources.
Will add more.
Thanks!

**Comment by CBID2** [comment]
> We are thinking to add a new `Category` for `YouTube` tutorials and `SubCategories` would be like `web developement` `frontend` `backend` `javascript` etc.
> 
> What are you takes on this? any suggestions?

I think you should @rupali-codes! :) People refer to this site a lot for tech education, so adding one is a great idea! 😄 👍🏾
